addon/models/tumblr-post-photo.js

```diff
@@ -0,0 +1,12 @@
+import DS from 'ember-data';
+import Post from './tumblr-post';
+import { alias } from '@ember/object/computed'
+
+const attr = DS.attr;
```

Comment:
You can import this directly (I haven't posted the entire project over, but no reason not to use it now):

`import attr from 'ember-data/attr';`